lovesay/love.py

```diff
@@ -134,11 +119,11 @@ def main(quote, color_name):
 
     # Setting up the things needed for the output
     if quote is None:
-        raw_quote = generate_quote(from_file=True, file_path=get_file_path())
+        raw_quote = generate_quote(from_file=True, file_path=get_file_path(), max_width=max_width)
     else:
-        raw_quote = generate_quote(from_file=False, quote=quote)
+        raw_quote = generate_quote(from_file=False, quote=quote, max_width=max_width)
```

Comment:
With the change mentioned above, this could be cleaned up as just a positional argument instead of a keyword one.

Reply:
I can move this to the first argument as a positional argument (assuming `generate_quote()` keeps max_width as the first argument), but not at the end. 
```
  File "/usr/local/lib/python3.9/site-packages/lovesay/love.py", line 123
    raw_quote = generate_quote(from_file=False, quote=quote, max_width)
                                                                      ^
SyntaxError: positional argument follows keyword argument
```

Reply:
With the above discussion in mind, I think this makes sense too!